scripts/eosio_build_amazon.sh

```diff
@@ -16,8 +16,8 @@
 	printf "\tDisk space total: ${DISK_TOTAL}G\n"
 	printf "\tDisk space available: ${DISK_AVAIL}G\n"
 
-	if [ $MEM_MEG -lt 4000 ]; then
-		printf "\tYour system must have 4 or more Gigabytes of physical memory installed.\n"
+	if [ $MEM_MEG -lt 7000 ]; then
```

Comment:
Didn't we settle on 8 GB of RAM during the scrum today?

Reply:
@gleehokie We explicitly and unquestionably decided in the presence of all that the minimum requirements for physical RAM would be 8 Gigs.
However ... my dev servers only have been 7.8 gigs of ram installed!
[bhamilton@ip-10-160-13-251 eos]$ free -m
              total        used        free      shared  buff/cache   available
Mem:           7821         193        5838          40        1788        7170
Swap:             0           0           0

Reply:
Can you please [update Docker Requirement for 7GB](https://github.com/EOSIO/eos/blob/master/Docker/README.md#docker-requirement)?